### ISSUE TITLE
[SELF_HOSTING] Step 4

### ISSUE THREAD

**Issue by lazna** [idoco/intergram#123]
Setting-up self hosted instance of intergram, but when step 4 from manual reached

"Open a chat with your bot and hit /start to get your unique chat ID"

no response when put `/start` to my custom telegram bot. No response, no even error message, bot just do not respond by anything

**Comment by mkrasselt1** [comment]
no, you definitly need to start chatting with @Intergram first, without that, you cant set it up at all, (because ists nit already running)

**Comment by mdestafadilah** [comment]
You can follow instruction in my issue please.
https://github.com/idoco/intergram/issues/121#issue-1493738944
